VersionControlUI/CMakeLists.txt

```diff
@@ -8,8 +8,10 @@ add_library(VersionControlUI SHARED
 	src/VersionControlUIException.h
 	src/versioncontrolui_api.h
 	src/VersionControlUIPlugin.h
+	src/DiffManager.h
```

Comment:
This commit is supposed to add a dependency, but you are adding here additional source files. This is not right, adding these files should have been done in a different commit or you should have named the commit appropriately.